Editor/SnapZone/SnapZoneSettings.cs

```diff
@@ -35,11 +35,22 @@ public class SnapZoneSettings : ScriptableObject
         /// </summary>
         [Tooltip("This color is used when a valid object is hovering the snap zone.")]
         public Color ValidationColor = new Color32(0, 255, 0, 50);
+        
+        /// <summary>
+        /// This color is used when an <see cref="InteractableObject"/> is hovering a <see cref="SnapZone"/>.
+        /// </summary>
+        [Tooltip("This color is used when a valid object is hovering the snap zone.")]
```

Comment:
Is this the real behavior? If so, what is the difference between `InvalidColor` and  `ValidationColor`?

Reply:
fixed